grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -504,8 +537,12 @@ def main():
                          "when -{} flag given").format('p'))
         importer.print_products()
         return 0
-    
+
     importer.import_products(flags['r'], flags['l'], flags['o'])
+
+    if options['footprints']:
+        importer.set_footprint(options['footprints'])
```

Comment:
Could you include set_footprint() into import_products()? If the footprint is used to limit the computational region, that can speed-up projection (r.import) quite a bit and maybe also import without projection (r.in.gdal), especially for small footprints. So, import procedure would be:
 1. Set computational region to footprint (if provided and not options["extent"] == "region", in the latter case the overlapping area could be used as temp region)
 2. import
 3. Clip to footprint if provided
 4. round if import is done with r.import (r.proj) (See: #127)